### PR TITLE
Please review security changes / error.

### PR DESCRIPTION
**Pull request by joesoftware**
Receiving the following error during submission to REST:

``` javascript
{
    "error": {
        "name": "Error",
        "status": 500,
        "message": "unable to verify the first certificate",
        "code": "UNABLE_TO_VERIFY_LEAF_SIGNATURE",
        "stack": "Error: unable to verify the first certificate\n    at Error (native)\n    at TLSSocket.<anonymous> (_tls_wrap.js:1003:38)\n    at emitNone (events.js:67:13)\n    at TLSSocket.emit (events.js:166:7)\n    at TLSSocket._finishInit (_tls_wrap.js:570:8)"
    }
}
```

### CODE REVIEW

File: hca-api-stub/voa-rest.js [hunk 1]
```diff
@@ -8,14 +8,26 @@ function attach(app) {
...
     {
       connector: require('loopback-connector-soap'),
       remotingEnabled: true,
       wsdl: path.join(__dirname, './voa.wsdl'),
-      url: endpoint.e8432
+      url: endpoint.preprod,
+      security: {
+        scheme: 'ClientSSL',
+        certPath: path.join(__dirname, './healthcare.application.crt'),
+        keyPath: path.join(__dirname, './healthcare.application.key')
```
Comment (awong-dev) [inline hca-api-stub/voa-rest.js]: If we do this, will the server start if the file is missing?

File: hca-api-stub/voa-rest.js [hunk 2]
```diff
@@ -8,14 +8,26 @@ function attach(app) {
...
+    prod: 'https://vaww.esr.aac.va.gov/voa/voaSvc'
   };
 
   const ds = loopback.createDataSource('soap',
     {
       connector: require('loopback-connector-soap'),
       remotingEnabled: true,
       wsdl: path.join(__dirname, './voa.wsdl'),
-      url: endpoint.e8432
+      url: endpoint.preprod,
```
Comment (robbiethegeek) [inline hca-api-stub/voa-rest.js]: This also seems like something that should be looped up in https://github.com/department-of-veterans-affairs/healthcare-application-team/issues/202

File: hca-rest-soap-connector/voa-rest.js [hunk 3]
```diff
@@ -6,12 +6,24 @@ var app = module.exports = loopback();
...
 
+// app.set('legacyExplorer', false); TODO should this be set? In production, we don't want exporer.
+
 var ds = loopback.createDataSource('soap',
   {
     connector: require('loopback-connector-soap'),
     remotingEnabled: true,
-    // wsdl: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc?wsdl' // The url to WSDL
-    wsdl: path.join(__dirname, './voa.wsdl')
+    url: 'https://vaww.esrpre-prod.aac.va.gov/voa/voaSvc',
```
Comment (robbiethegeek) [inline hca-rest-soap-connector/voa-rest.js]: Same here with reference to: https://github.com/department-of-veterans-affairs/healthcare-application-team/issues/202

File: hca-api-stub/voa-rest.js (hunk 1 above)
Comment (joesoftware) [inline hca-api-stub/voa-rest.js]: Nope. I'll add logic to exclude the security object if there are no certs. Better solutions are invited.

File: hca-rest-soap-connector/voa-rest.js (hunk 3 above)
Comment (joesoftware) [inline hca-rest-soap-connector/voa-rest.js]: Thanks Robbie. Even though I made changes to this directory, let's sunset review of those files.

File: hca-api-stub/voa-rest.js (hunk 2 above)
Comment (joesoftware) [inline hca-api-stub/voa-rest.js]: Good point. For the short-term, I'll conditionally include the server info based on availability of security files.

File: hca-api-stub/voa-rest.js [hunk 4]
```diff
@@ -1,21 +1,83 @@
+const fs = require('fs');
 const loopback = require('loopback');
 const path = require('path');
 
+// TODO: node.js did not like `securityArtifacts` being declared as `let`. Why?
```
Comment (awong-dev) [inline hca-api-stub/voa-rest.js]: Sigh. cause we need to enable strict mode or something weird. TODO is good enough for now.

Comment (robbiethegeek) [inline hca-api-stub/voa-rest.js]: yeah this is another version related issue, the support for let was hidden behind strict in the initial LTS release 4.2.x, and now that is not the case.

**Comment by awong-dev** [comment]
LGTM

**Pull request merged**